primitives/std.lib

```diff
@@ -822,3 +822,57 @@ primitive fixed_p_std_div[width, int_width, fract_width](left: width, right: wid
 
     }
 }
+
+// the bigger integer bit always comes left, one with bigger fractional bit comes right
+primitive fixed_p_std_add_dbit[width, int_width1, fract_width1, int_width2, fract_width2, out_width] (left: width, right: width) ->(out: out_width){
```

Comment:
super small formatting note: can you format the signature so that it's `primitive name[p1, p2, ..](l: w, r: w) -> (out: w) {`